R/fd_remote_sensing.R

```diff
@@ -99,23 +100,25 @@ fd_ceptometer <- function() {
   cept <- subset(cept, cept$project == "forte")
 
   # Rename columns
-  names(cept)[names(cept) == "Average.Above.PAR"] <- "aPAR"
-  names(cept)[names(cept) == "Average.Below.PAR"] <- "bPAR"
-  names(cept)[names(cept) == "Leaf.Area.Index..LAI."] <- "LAI_cept"
+  names(cept)[names(cept) == "Average.Above.PAR"] <- "a_par"
+  names(cept)[names(cept) == "Average.Below.PAR"] <- "b_par"
+  names(cept)[names(cept) == "Leaf.Area.Index..LAI."] <- "lai_cept"
 
   # Make DateTime column as datetime object
-  cept$Timestamp <- as.POSIXct(cept$DateTime, format = "%m/%d/%Y %H:%M")
+  cept$timestamp <- as.POSIXct(cept$DateTime, format = "%m/%d/%Y %H:%M")
```

Comment:
Might be good to set a time zone here (and elsewhere?). Is this local time or UTC?